Nos/Assets/Localization/ImagePicker.xcstrings

```diff
@@ -176,6 +176,39 @@
         }
       }
     },
+    "getAccount" : {
+        "extractionState" : "manual",
+        "localizations" : {
+          "en" : {
+            "stringUnit" : {
+              "state" : "translated",
+              "value" : "Get Account"
+            }
+          }
+        }
+      },
+      "errorUploadingFileExceedsSizeLimit" : {
+            "extractionState" : "manual",
+            "localizations" : {
+              "en" : {
+                "stringUnit" : {
+                  "state" : "translated",
+                  "value" : "Get a Nostr.Build account to upload larger files"
+                }
+              }
+            }
+      },
+      "errorUploadingFileExceedsLimit" : {
+          "extractionState" : "manual",
+          "localizations" : {
+            "en" : {
+              "stringUnit" : {
+                "state" : "translated",
+                "value" : "The max file size for free media uploads is of %@. To upload large files, upgrade to a Nostr.Build Professional account."
+              }
+            }
+          }
+    },
```

Comment:
This one doesn't seem to be used, but I might be missing something. Did something get missed, perhaps, or is this just unneeded?

Reply:
Oh yes, i tried to add that then got stuck and walked back out util i found the bug and forgot to add it back. I've got it added back now.